### PR TITLE
Fixed code style for JSON blocks and styles of the collapsible elements

### PR DESCRIPTION
**Pull request by andrewbess**
## Purpose of this pull request

This pull request (PR) fixes code style of the JSON blocks, headers from "[Order processing tutorial](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-intro.html)", and styles of the collapsible elements

## Affected DevDocs pages

-  [Step 2. Get the admin token](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-admin-token.html)
-  [Step 3. Create a customer](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-create-customer.html)
-  [Step 4. Create a quote](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-create-quote.html)
-  [Step 5. Add items to the cart](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-add-items.html)
-  [Step 6. Prepare for checkout](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-prepare-checkout.html)
-  [Step 7. Create an order](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-create-order.html)
-  [Step 8. Create an invoice](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-create-invoice.html)
-  [Step 9. Create a shipment](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-create-shipment.html)
-  [Step 10. Issue a partial refund](https://devdocs.magento.com/guides/v2.3/rest/tutorials/orders/order-issue-refund.html)

## Stylesheet fix
Previously:
previously

Now:
now

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/rest/tutorials/orders/order-add-items.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ The following example adds an orange medium-sized Radiant women's t-shirt (`sku`
 
 **Headers:**
 
-`Content-Type` `application/json`
+`Content-Type`: `application/json`
 
-`Authorization` `Bearer <customer token>`
+`Authorization`: `Bearer` `<customer` `token>`
```
Comment (dobooth) [inline src/guides/v2.3/rest/tutorials/orders/order-add-items.md]: No need to split <customer token>. Please revert them.

Comment (andrewbess) [reply src/guides/v2.3/rest/tutorials/orders/order-add-items.md]: Hello @dobooth.
Thank you for your remark.
I fixed it.
Please check.
Thank you in advance.

**Comment by dobooth** [comment]
running tests

**Pull request merged**